lib/src/dexode/EventBus.cpp

```diff
@@ -70,7 +72,7 @@ eventbus::stream::EventStream* EventBus::findStream(
 void EventBus::unlistenAll(const std::uint32_t listenerID)
 {
 	std::shared_lock readGuard{_mutexStreams};
-	for(auto& eventStream : _eventToStream)
+	for(const auto& eventStream : _eventToStream)
```

Comment:
It doesn't change anything but it is easier to understand the code because we modify something. Not important.